src/index.html

```diff
@@ -1,13 +1,498 @@
+              form.addEventListener('submit', (e) => { e.preventDefault();
+              form.reset();
+              });
+            </script>
+          </div>
+        </div>
+      </section>
+    </main>
+
+    <footer class="footer">
+      <div class="grid grid--tablet footer__grid">
+        <div class="footer__top">
+          <a href="#" class="header__logo">
+            NA
+            <br>
+            MU
+          </a>
+
+          <div class="footer__icon-container">
+            <a
+            href="https://ru-ru.facebook.com/namu.museum/"
+            target="_blank"
+            class="footer__icon"
+            >
+              <img
+              src="./images/facebook-icon-1.svg" alt="Facebook"
+              class="footer__icon--inst"
+              >
+            </a>
+            <a
+            href="https://www.instagram.com/namu.museum/"
+            target="_blank"
+            class="footer__icon"
+            >
+              <img
+              src="./images/instagram-glyph-1.svg"
+              alt="Instagram"
+              class="footer__icon--inst"
+              >
+            </a>
+          </div>
+        </div>
+
+        <div class="footer__container-2 grid__item--tablet-2-5 grid__item--desktop-3-6">
+          <div class="footer__schedule">
+            <h2 class="footer__schedule-title">
+              Графік роботи
+            </h2>
+            <p class="footer__schedule-content">
+              12:00 — 20:00 середа
+              <br>
+              12:00 — 20:00 четвер
+              <br>
+              12:00 — 20:00 п’ятниця
+              <br>
+              11:00 — 19:00 субота
+              <br>
+              11:00 — 19:00 неділя
+            </p>
+            <p class="footer__schedule-weekend">
+              Вихідні: понеділок, вівторок
+              <br>
+              Каса припиняє роботу за 1 годину до
+              <br>
+              закриття музею
+            </p>
+          </div>
+
+          <div class="footer__contacts">
+            <h2 class="footer__contacts-title">
+              Контакти:
+            </h2>
+            <div class="footer__contacts-content" id="contacts">
+              <a
+              href="https://goo.gl/maps/PKcrPA7tbiRY1tLE6"
+              class="footer__contacts-infolink"
+              target="_blank"
+              >
+                вул. М.
+                <br>
+                Грушевського, 6;
+                <br>
+                Київ, 01001
+              </a>
+              <br>
+              <a
+                href="tel:+38 (044) 278-74-54"
+                class="footer__contacts-infolink"
+                target="_blank"
+              >
+                тел. +38 (044)
+                <br>
+                278-13-57
+              </a>
+              <a
+                href="fax:+38 (044) 278-74-54"
+                class="footer__contacts-infolink"
+                target="_blank"
+              >
```

Comment:
use phone numbers in attributes without additional symbols like space or round brackets. Just `tel:+380442787454`